dss_client/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 # Set the minimum required version of CMake for this project.
-cmake_minimum_required(VERSION 3.7)
+#cmake_minimum_required(VERSION 3.7)
+cmake_minimum_required(VERSION 3.9.4)
 
 # Set this project's name.
 project("dss")
 
 # Set the C++ standard to use to build this target.
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Why we need c++17 ?

Reply:
That came from the elbencho's requirement